### ISSUE TITLE
422 - Validation Failed When using GithubPullRequestFormatter

### ISSUE THREAD

**Issue by samnang** [mmozuras/pronto#170]
Here is the output when I run:

```
Failed to post: POST https://api.github.com/repos/user_a/repo1/pulls/49/comments: 422 - Validation Failed
Error summary:
  resource: PullRequestReviewComment
  code: invalid
  field: position // See: https://developer.github.com/v3/pulls/comments/#create-a-comment
66 Pronto messages posted to GitHub
Failed to post: POST https://api.github.com/repos/user_a/repo1/pulls/49/comments: 422 - Validation Failed
Error summary:
  resource: PullRequestReviewComment
  code: invalid
  field: position // See: https://developer.github.com/v3/pulls/comments/#create-a-comment
66 Pronto messages posted to GitHub
```

When I look at Github API documentation, it seems `position` is required, but it seem it's missing when creating comment here https://github.com/mmozuras/pronto/blob/809ed1d50be1632fa2f3885eb13d1ff865158a0e/lib/pronto/formatter/git_formatter.rb#L66-L72

**Comment by jglapa** [comment]
Getting the exact same error!

**Comment by jglapa** [comment]
looks like duplicate to #149

**Comment by mmozuras** [comment]
Fixed and released [v0.7.1](https://github.com/mmozuras/pronto/releases/tag/v0.7.1).

**Comment by samnang** [comment]
@mmozuras thanks for fixing it. Love it!
